DKP-184 Desktop 4.9.0 release notes (#14863)
docker/docker.github.io: desktop/release-notes.md

> .button .primary-btn }
> [Arch (experimental)](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.9.0-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
> .button .primary-btn }

### New

- Added a footer to the Docker Dashboard with general information about the Docker Desktop update status and Docker Engine statistics
- Re-designed the containers table, adding:
  - A button to copy a container ID to the clipboard
  - A pause button for each container
  - Column resizing for the containers table
  - Persistence of sorting and resizing for the containers table
  - Bulk deletion for the containers table

### Upgrades

- [Compose v2.6.0](https://github.com/docker/compose/releases/tag/v2.6.0)
- [Docker Engine v20.10.16](https://docs.docker.com/engine/release-notes/#201016)
- [containerd v1.6.4](https://github.com/containerd/containerd/releases/tag/v1.6.4)
- [runc v1.1.1](https://github.com/opencontainers/runc/releases/tag/v1.1.1)
- [Go 1.18.2](https://github.com/golang/go/releases/tag/go1.18.2)

### Bug fixes and minor changes

#### For all platforms

> - Fixed Docker Desktop hanging when quitting while paused

Reviewer: ```suggestion
- Fixed an issue which caused Docker Desktop to hang if you quit the app whilst Docker Desktop was paused
```

- Fixed the Kubernetes cluster not resetting properly after the PKI expires
- Fixed the extensions' marketplace not using the defined http proxies
- Improved the Dashboard logs search to allow spaces

> - Fixed middle button click on links in the Dashboard to correctly open external links

Reviewer 2: @aevesdocker We are aware of few of them still not working as intended yet, is there any way to tamper a bit that line and the one below it ?

> - Fixed middle button click on links in the Dashboard to correctly open external links

Author: Maybe we could just drop these until we truly fix everything?
Apparently it’s not a user reported issue, is it?

> - Fixed middle button click on links in the Dashboard to correctly open external links

Author: I dropped that one, but kept the other one.

- Fixed middle button click on buttons on the Dashboard to behave as a left button click instead of opening a blank window

#### For Mac

> - Avoid creating `/opt/containerd/bin` and `/opt/containerd/lib` on the host if `/opt` has been added to the file sharing directories list

Reviewer: ```suggestion
- Fixed an issues which avoids creating `/opt/containerd/bin` and `/opt/containerd/lib` on the host if `/opt` has been added to the file sharing directories list
```

Author: Changed to 
> Fixed an issue to avoid creating `/opt/containerd/bin` and `/opt/containerd/lib` on the host if `/opt` has been added to the file sharing directories list.

as it’s the other way around: the issue was that it created these folders, and we don’t want that.

#### For Windows

> - Fixed a bug in the WSL 2 integration where if a file or directory is bind-mounted to a container, and the container exits, then the file or directory is replaced with the other type of object with the same name i.e. a file is replaced with a directory or a directory with a file, attempts to bind-mount the new object failed

Reviewer: ```suggestion
- Fixed a bug in the WSL 2 integration where if a file or directory is bind-mounted to a container, and the container exits, then the file or directory is replaced with the other type of object with the same name. For example, if a file is replaced with a directory or a directory with a file, any attempts to bind-mount the new object fails.
```
